test/uk/gov/hmrc/hecstubs/controllers/IndividualAccountOverviewControllerSpec.scala

```diff
@@ -51,12 +53,57 @@ class IndividualAccountOverviewControllerSpec extends AnyWordSpec with Matchers
 
     "fetching account overview details " should {
 
-      "return 200 and return the correct response" in {
+      "return 200 with correct status" when {
+        "utr starts with 1111" in {
```

Comment:
can we have one more test, if UTR starts with 1111 but don't have 10 chars? please ignore if not relevant

Reply:
sure, I'll add